### ISSUE TITLE
Move Maven packaging to the winsw organization

### ISSUE THREAD

**Issue by oleg-nenashev** [winsw/winsw#481]
### Problem statement

In previous releases I detached Maven packaging from the WinSW repo and moved it to https://github.com/jenkinsci/winsw-maven-packaging (#332) . Since we have a GitHub organization now, it may make sense to move the repo to the organization.

Feedback would be appreciated

**Comment by NextTurn** [comment]
Maven releases can be automated as part of Azure Pipelines CD.

**Comment by oleg-nenashev** [comment]
Yes, it makes sense. In such case we should probably also consider deploying the releases to Maven Central and changing groupId of Maven packages

**Comment by oleg-nenashev** [comment]
Though I wonder how many WinSW users actually need it. Maybe it is easier to deprecate Maven packaging and to document how to use WinSW directly from GitHub releases

**Comment by NextTurn** [comment]
How can Jenkins (and its modules) fetch WinSW without Maven packages?

**Comment by oleg-nenashev** [comment]
There are Maven plugins for GitHub releases. Also, it is quite trivial to get fetching working without any special plugins, e.g. similarly to how https://github.com/jenkinsci/winsw-maven-packaging/blob/winsw-2.4.0/pom.xml#L61-L82 works. Cons: Such approach would not allow tools like Dependabot to operate properly

**Comment by jeacott1** [comment]
@oleg-nenashev 
> Though I wonder how many WinSW users actually need it. Maybe it is easier to deprecate Maven packaging and to document how to use WinSW directly from GitHub releases

I am using winsw and maven, and would be very disappointed if it was suddenly not available.

**Comment by oleg-nenashev** [comment]
Thanks for the feedback @jeacott1 ! If there are active users, we should definitely keep it

**Comment by billbarni** [comment]
I am also using WinSW via Maven... I am trying to use the latest versions, but I was only able to find version 2.9.0, is there any updates on this?

**Comment by oleg-nenashev** [comment]
I'd guess the ball is on my side. Will try to release the maben package
tomorrow

**Comment by billbarni** [comment]
I'm still using 2.9.0 because the maven release never happened. Any news on this?
